torch/csrc/autograd/init.cpp

```diff
@@ -108,6 +148,11 @@ static PyObject * is_anomaly_mode_enabled(PyObject* _unused, PyObject *arg) {
 static PyMethodDef methods[] = {
   {"set_grad_enabled", (PyCFunction)set_grad_enabled, METH_O, nullptr},
   {"is_grad_enabled", (PyCFunction)is_grad_enabled, METH_NOARGS, nullptr},
+  {"set_autocast_enabled", (PyCFunction)set_autocast_enabled, METH_O, nullptr},
```

Comment:
Out of curiosity, what is the justification to bind this to the autograd and not general torch?

Reply:
It was convenient and no one minded.  I can move that in the mainline if this is not the appropriate home.

Reply:
But does this flag actually changes anything in the autograd? I can't see any change in there? Or does it only change aten ops?

Reply:
The calls change flags in aten/core/ATen/autocast_mode.cpp.  They enable or disable a custom DispatchKey (AutocastTensorId).  For the Python binding, I was imitating grad_mode because it seemed most similar to the minimal exposure I wanted.